lambda_functions/va_profile/va_profile_opt_in_out_lambda.py

```diff
@@ -41,21 +41,16 @@
 logger = logging.getLogger("VAProfileOptInOut")
 logger.setLevel(logging.DEBUG)
 
-CERTIFICATE_ARN = os.getenv("CERTIFICATE_ARN")
-OPT_IN_OUT_QUERY = """SELECT va_profile_opt_in_out(%s, %s, %s, %s, %s);"""
+ALB_CERTIFICATE_ARN = os.getenv("ALB_CERTIFICATE_ARN")
+ALB_PRIVATE_KEY_PATH = os.getenv("ALB_PRIVATE_KEY_PATH")
 NOTIFY_ENVIRONMENT = os.getenv("NOTIFY_ENVIRONMENT")
-ALB_PRIVATE_KEY_PATH = os.getenv("PRIVATE_KEY_PATH")
-# TODO - Make this an SSM call.
-SQLALCHEMY_DATABASE_URI = os.getenv("SQLALCHEMY_DATABASE_URI")
+OPT_IN_OUT_QUERY = """SELECT va_profile_opt_in_out(%s, %s, %s, %s, %s);"""
 VA_PROFILE_DOMAIN = os.getenv("VA_PROFILE_DOMAIN")
 VA_PROFILE_PATH_BASE = "/communication-hub/communication/v1/status/changelog/"
 
 
 if NOTIFY_ENVIRONMENT is None:
-    # Without this value, this code cannot know the path to the required
-    # SSM Parameter Store values and other resources.
-    sys.exit("NOTIFY_ENVIRONMENT is not set.")
-
+    sys.exit("NOTIFY_ENVIRONMENT is not set.  Check the Lambda console.")
```

Comment:
Does this print to the cloudwatch logs? If not we need a `logger` call.

Reply:
Yes, it does.  I have seen sys.exit statements in the logs before.